app/models/micropost.rb

```diff
@@ -0,0 +1,17 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  default_scope{order("created_at DESC")}
```

Comment:
```suggestion
  default_scope{order created_at: :DESC}
```

cơ mà ko dùng default scope nhé, dùng scope đi em

newest/ lastest ...hoặc sort_by_created_at